code_schemes/s03e21_yes_no_amb.json

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
Isn't this redundant due to s03e21.json?

Reply:
For Imaqal we are still imputing yes, no, amb codes separately. We only upgraded the reasons coding schemes to include yes, no prefix to aid in labeling.

Reply:
Oh ok. I thought you had switched to yes/no prefixed schemes only for new schemes to avoid all the problems with having two. But I think I was misunderstanding